sycl/test/graph/graph-explicit-dotp-buffer.cpp

```diff
@@ -24,9 +24,7 @@ int main() {
   float beta = 2.0f;
   float gamma = 3.0f;
 
-  sycl::property_list properties{
-      sycl::property::queue::in_order{},
-      sycl::ext::oneapi::property::queue::lazy_execution{}};
+  sycl::property_list properties{sycl::property::queue::in_order{}};
```

Comment:
think we can get rid of `in_order` here too, we only ever submit one thing to the queue, the exec graph